scratch-arduino-extensions/scripts/patch-gui.js

```diff
@@ -45,3 +46,22 @@ for (const ExtId of ExtIds) {
     }
 }
 
+
+
+let replacementListPath = path.join(__dirname, 'include-msgs.json');
+let includeMsgs = JSON.parse(fs.readFileSync(replacementListPath, 'utf8'));
+
+let fileContent = fs.readFileSync(EditorMessagesDir, 'utf8');
+let match = fileContent.match(/export default (\{.*\});/s);
+if (!match) {
+    throw new Error('Could not find object in file');
+}
+let obj = eval('(' + match[1] + ')');
+for (let lang in includeMsgs) {
+    console.log(`Adding "${lang}" translation`);
+    for (let key in includeMsgs[lang]) {
+        obj[lang][key] = includeMsgs[lang][key];
+    }
```

Comment:
you can use spread operation instead of this for loop
```suggestion
    obj[lang] = { ...obj[lang], ...includeMsgs[lang] };
```